confidential-data-hub/hub/tests/image-pull.rs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) 2024 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::time::Duration;
+
+use log::info;
+
+/// Script for preparing resources.json for offline-fs-kbc.
+const OFFLINE_FS_KBC_RESOURCE_SCRIPT: &str =
+    "../../image-rs/scripts/install_offline_fs_kbc_files.sh";
+
+#[rstest::rstest]
+#[case("busybox:latest", "unsigned unencrypted image")]
+#[case(
+    "ghcr.io/confidential-containers/test-container-image-rs:cosign-signed",
+    "cosign signed unencrypted image"
+)]
+#[case(
+    "ghcr.io/confidential-containers/test-container:encrypted",
+    "unsigned encrypted image"
+)]
+#[case("quay.io/liudalibj/private-busy-box", "private registry")]
+#[tokio::test]
+#[serial_test::serial]
+async fn test_pull_image(#[case] image_ref: &str, #[case] item: &str) {
```

Comment:
Would it be cleaner to use [case descriptions](https://docs.rs/rstest/latest/rstest/attr.rstest.html#optional-case-description) here, rather than passing in `item` as that e.g.
```
#[case("busybox:latest", "unsigned unencrypted image")]
```
becomes
```
#[case::unsigned_unencrypted_image("busybox:latest")]
```
?

Reply:
Oh! THis is a really cool feature! Let me do this!

Reply:
Yeah, I need to remember to use it more myself too!